apps/parsley/src/analytics/loadingPage/useLogDownloadAnalytics.ts

```diff
@@ -3,19 +3,19 @@ import { LogTypes } from "constants/enums";
 
```

Comment:
Were the events in `useLogWindowAnalytics` and `usePreferencesAnalytics` skipped intentionally?